assets/_sass/custom-viz/boeing/_base.scss

```diff
@@ -18,6 +18,26 @@
 
     svg {
       height: 100%;
+      width: 100%;
+    }
+
+    .counter {
+      p {
+        font-family: $font_inconsolata;
+        text-transform: uppercase;
+        margin: 0rem 1.5rem;
+
+        span {
+          font-weight: bold;
+        }
+      }
+
+      @include breakpoint("large") {
+        display: flex;
+        justify-content: flex-end;
+        margin: 0 auto 1rem;
+        width: 70%;
```

Comment:
Rather than setting a width of 70%, I would set a max-width of 700px (the same as the text container). That way you know they will line up. This would be a good use case of making a new variable to keep those values consistent.